src/loaders/bisheng-data-loader.js

```diff
@@ -49,9 +49,12 @@ module.exports = function bishengDataLoader(/* content */) {
     });
   }
 
-  return 'module.exports = {' +
+  const content = 'module.exports = {' +
     `\n  markdown: ${markdownData.stringify(markdown, themeConfig.lazyLoad, isSSR)},` +
     `\n  plugins: [\n${pluginsString}\n],` +
     `\n  picked: ${JSON.stringify(picked, null, 2)},` +
     `\n};`;
+  // fs.writeFileSync('./markdown-tree.debug', content);
```

Comment:
Actually, you should write a test case to test this content, instead of inspect by yourself...